frontend/src/concepts/pipelines/content/compareRuns/metricsSection/markdown/useFetchMarkdownMaps.ts

```diff
@@ -39,18 +39,27 @@ const useFetchMarkdownMaps = (
         .map(async (path) => {
           const { run } = path;
           let sizeBytes: number | undefined;
-
-          const url = await getStorageObjectUrl(path.linkedArtifact.artifact).catch(
-            () => undefined,
-          );
+          let url: string | undefined;
+          if (
+            path.linkedArtifact.artifact.getType() === ArtifactType.MARKDOWN ||
+            path.linkedArtifact.artifact.getType() === ArtifactType.HTML
+          ) {
+            url = await getStorageObjectRenderUrl(path.linkedArtifact.artifact).catch(
+              () => undefined,
+            );
+          } else {
+            url = await getStorageObjectDownloadUrl(path.linkedArtifact.artifact).catch(
```

Comment:
you dont need this part. as it will just auto download if put in a iframe

Reply:
Yep you are right, removed.